src/View/ViewBuilder.php

```diff
@@ -52,14 +52,14 @@ class ViewBuilder implements JsonSerializable, Serializable
     /**
      * The plugin name to use.
      *
-     * @var string|null|false
+     * @var string|null
      */
     protected $_plugin;
 
     /**
      * The theme name to use.
      *
-     * @var string|null|false
+     * @var string|null
```

Comment:
sure?
also, below _layout can also be null (or false if no layout should be used), right?

Reply:
I am fixing the wrong layout one in https://github.com/cakephp/cakephp/pull/12588 then in master.